telegram/ext/jobqueue.py

```diff
@@ -208,22 +241,39 @@ class Job(object):
         callback (function): The callback function that should be executed by the Job. It should
             take two parameters ``bot`` and ``job``, where ``job`` is the ``Job`` instance. It
             can be used to terminate the job or modify its interval.
-        interval (float): The interval in which this job should execute its callback function in
-            seconds.
+        interval (float or datetime.time): The interval in which this job should execute its
+            callback function in seconds.
         repeat (Optional[bool]): If this job should be periodically execute its callback function
             (``True``) or only once (``False``). Defaults to ``True``
         context (Optional[object]): Additional data needed for the callback function. Can be
             accessed through ``job.context`` in the callback. Defaults to ``None``
+        days (Tuple): Defines on which days the job should be ran.
 
     """
     job_queue = None
 
-    def __init__(self, callback, interval, repeat=True, context=None):
+    def __init__(self,
+                 callback,
+                 interval,
+                 repeat=True,
+                 context=None,
+                 days=tuple(day for day in Days)):
         self.callback = callback
         self.interval = interval
         self.repeat = repeat
         self.context = context
 
+        if not isinstance(days, tuple):
+            err_msg = "The 'days' argument should be of type 'tuple'"
```

Comment:
What is the max line length for the project? I use 84 as my personal standard, so when a message is too long, I use a variable.